docs/guides.rst

```diff
@@ -67,8 +67,6 @@ You can also find out which input files are required for a given preparation sta
      - free_preequil.prm7, free_preequil.rst7
      - The solvated ligand after heating and short initial equilibration steps
```

Comment:
Please could you add a short section on SOMD config options? And possibly how to update them (if you add this functionality)?